### DISCUSSION TITLE
"Marked as answer" banner artwork is a little blurry on iPad Pro 12.9-inch

### DISCUSSION THREAD

**Discussion by AustinConlon** [community/community#10817] [Mobile] [answered]
![IMG_0002](https://user-images.githubusercontent.com/1305957/151826337-305f628f-c567-44e8-98be-be85aa6e81cd.png)

**Comment by eliperkins** [comment, marked as answer]
Ah, great eyes. This is definitely a bug, thanks a ton for reporting this. 💖 

I've filed an issue on our repo to get this addressed. I'll follow up with you once this bug fix ships!
